### PR TITLE
Drop EL7 from the nightly quickstart installation guide

### PR DESCRIPTION
**Pull request by upadhyeammit**
Cherry-pick into:
None

### CODE REVIEW

File: guides/common/modules/proc_configuring-repositories.adoc [hunk 1]
```diff
@@ -6,9 +6,8 @@ Use this procedure to enable the repositories that are required to install {Prod
 
 ifdef::foreman-el,katello[]
 * xref:#repositories-centos-8[CentOS 8]
-* xref:#repositories-centos-7[CentOS 7]
 endif::[]
-ifdef::foreman-el,katello,satellite[]
+ifdef::satellite[]
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-repositories.adoc:10]: I'm not sure this is needed. It's not like Satellite is going to support EL7 either.

**Review by ekohl — COMMENTED** [review]
I had hoped https://github.com/theforeman/foreman-documentation/pull/982 would first get attention since it makes this easier. I also think that we should assume Satellite also drops RHEL 7 support. This does imply that everywhere it must be RHEL 8.

It may make https://github.com/theforeman/foreman-documentation/pull/1213 obsolete and we can simplify it by always using the EL 8 path.

File: guides/common/modules/proc_configuring-repositories.adoc (hunk 1 above)
Comment (upadhyeammit) [reply guides/common/modules/proc_configuring-repositories.adoc]: Removed!

**Review by melcorr — APPROVED** [review]
LGTM!

**Pull request merged**